pkg/printers/internalversion/printers.go

```diff
@@ -2233,6 +2241,26 @@ func printClusterTrustBundleList(list *certificates.ClusterTrustBundleList, opti
 	return rows, nil
 }
 
+func printPodCertificateRequest(obj *certificates.PodCertificateRequest, options printers.GenerateOptions) ([]metav1.TableRow, error) {
+	row := metav1.TableRow{
+		Object: runtime.RawExtension{Object: obj},
+	}
+	row.Cells = append(row.Cells, obj.Name, obj.Spec.PodName, obj.Spec.SignerName)
```

Comment:
does it make sense to indicate any status? whether it was issued? whether it is currently failed, etc?